team-norms/retrospectives.md

```diff
@@ -0,0 +1,66 @@
+# Retrospectives
+
+These retrospectives occur every 2 weeks on a Friday. There are 2 hours booked for this purpose. 
```

Comment:
I think there is a large block of time i.e. 9 days, then a slightly overloaded day. I believe it should be possible for people to work their commitments around this